### PR TITLE
Fix tool_render with direct import from pydantic

### PR DESCRIPTION
**Pull request by minmin-intel**
## Description

latest langchain structuredtool needs to import from pydantic. Old tool renderer gives error with tool args. Fix tool render by directly import from pydantic

## Issues

 `n/a`.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Dependencies

No newly introduced 3rd party dependency if exists.

## Tests

Unit test

### CODE REVIEW

File: comps/agent/langchain/src/strategy/ragagent/planner.py [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
 from langchain_core.output_parsers.openai_tools import PydanticToolsParser
 from langchain_core.prompts import PromptTemplate
-from langchain_core.pydantic_v1 import BaseModel, Field
 from langchain_huggingface import ChatHuggingFace, HuggingFaceEndpoint
 from langchain_openai import ChatOpenAI
 from langgraph.checkpoint.memory import MemorySaver
 from langgraph.graph import END, START, StateGraph
 from langgraph.graph.message import add_messages
 from langgraph.prebuilt import ToolNode, tools_condition
+from pydantic import BaseModel, Field
```
Comment (xuechendi) [inline comps/agent/langchain/src/strategy/ragagent/planner.py:17]: any specific reason for this change?

**Review by xuechendi — APPROVED** [review]
LGTM

**Comment by xuechendi** [comment]
@minmin-intel , may you also update the langchain version who need : from pydantic import BaseModel, Field; So we can avoid install old version which breaks the workflow

**Comment by minmin-intel** [comment]
> @minmin-intel , may you also update the langchain version who need : from pydantic import BaseModel, Field; So we can avoid install old version which breaks the workflow

We are installing latest langchain in requirement.txt 
image
Did you want to fix the version? @xuechendi

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**